prow/github/fakegithub/fakegithub.go

```diff
@@ -270,3 +270,7 @@ func (f *FakeClient) ListCollaborators(org, repo string) ([]github.User, error)
 	}
 	return result, nil
 }
+
+func LabelID(org, repo string, number int, label string) string {
```

Comment:
Not sure we really need an exported func for this -- saves a couple chars but at the same time we're unlikely to move every use of the `%s/%s#%d:%s` usage over.